js/components/meta/about-dialog.vue

```diff
@@ -6,7 +6,7 @@
 
     <main>
       <div class="logo">
-        <img src="~#/../img/logo.svg" width="128" height="auto">
+        <img src="../../../img/logo.svg" width="128" height="auto">
```

Comment:
Is this change necessary?

Reply:
If I leave it with `~#/../img/logo.svg`, webpack errors with the following line:
```
Module not found: Error: Can't resolve 'null' in 'koel/resources/assets/js/components/meta'
```

If I remove the tilde `#/../img/logo.svg`, I get a 404 error :confused: 
It looks like it's necessary, I don't what configuration to change. Where does the tilde `~`  point to by the way ? I have not been able to find out what it means.

Reply:
Looks like there's a change in how webpack resolves `~`, so now we don't need it anymore. If we replace `~#` with only `#` (or `@` if we prefer, as both `@/../` and `#/../` will resolve into `assets/`), the code will work.

Reply:
OK thanks for the info 🙂. I tried with # but it is interpreted as an html anchor link, so it led to 404.
I'll try with @

Reply:
It works with `@/../img/logo.svg`